src/test/kotlin/com/terraformation/backend/seedbank/search/SearchServiceFuzzySearchTest.kt

```diff
@@ -43,4 +43,20 @@ internal class SearchServiceFuzzySearchTest : SearchServiceTest() {
 
     assertEquals(expected, result)
   }
+
+  @Test
+  fun `fuzzy search on text fields tries exact matching first`() {
+    accessionsDao.update(accessionsDao.fetchOneById(AccessionId(1000))!!.copy(number = "22-1-100"))
+    accessionsDao.update(accessionsDao.fetchOneById(AccessionId(1001))!!.copy(number = "22-1-101"))
+
+    val fields = listOf(accessionNumberField)
+    val searchNode = FieldNode(accessionNumberField, listOf("22-1-100"), SearchFilterType.Fuzzy)
```

Comment:
in this case, would a fuzzy search also return `22-1-101`?

Reply:
> in this case, would a fuzzy search also return `22-1-101`?

Yes. I'll update the test to make that more explicit.